### DISCUSSION TITLE
dnd off command is not working for me

### DISCUSSION THREAD

**Discussion by d1jpd01** [ttscoff/bunch-docs#100] [Q&A] [answered]
Here are the commands in my first Bunch.

@@ 

(dnd) # Do not disturb mode.

Zoom

!(dnd off)

This works fine when I open the Bunch, however, when I close the Bunch dnd does not turn off.  Am I doing something wrong?  
I am using Bunch 1.4.1 on a MacBook Air (M1) running Big Sur 11.4.

**Comment by ttscoff** [comment]
DND should be toggled back automatically when closing the bunch, without the ! command. If you leave it out, does it behave as expected?

**Reply by ttscoff** [reply]
That is, unless you're in Single Bunch Mode and the next Bunch that opens also has a DND command, in which case DND is ignored when closing the previous Bunch.

**Reply by ttscoff** [reply]
Ok, this just started happening for me as well, with DND being left on after closing a Bunch that enabled it. I'll have to dig in to see what might have changed.

**Reply by manuelhubacher** [reply]
Good to know that I'm not the only one experiencing this. The strange thing is that I have to click multiple times on the DND icon in Control Center in order to turn off DND. This only happens after I activated DND using Bunch. 

(macOS 11.4, Bunch Beta Version 1.4.1)

**Reply by ttscoff** [reply]
This is officially fixed in the beta, planning to release to the stable channel today.

**Comment by ttscoff** [comment]
Same here. Not sure why it worked before and isn't now, will have to do 
some digging.

**Comment by ttscoff** [comment, marked as answer]
Ok, the fix for this has officially been released to the main channel (v1.4.3).

**Reply by splottdodger** [reply]
Maybe I am doing something stupid, but this is still an issue for me in v1.4.4. What might I have got wrong?

**Reply by ttscoff** [reply]
Just double checked and it's working for me on macOS 11. What OS are you on?

**Comment by splottdodger** [comment]
I have macOS 11.5.2.

**Comment by ttscoff** [comment]
I don't have an answer for you, since the fix a couple versions ago it's 
been working better than ever for me, and I haven't heard anyone else 
mention it since then. That's frustrating that it's still happening for 
you.

-Brett
